remi/gui.py

```diff
@@ -537,7 +538,7 @@ def set_on_mousedown_listener(self, listener, funcname):
     def onmouseup(self, x, y):
         """Called when the user releases left or right mouse button over a Widget.
         """
-        return self.eventManager.propagate(self.EVENT_ONMOUSEUP, [x, y])
+        return self.eventManager.propagate(self.EVENT_ONMOUSEUP, (x, y))
```

Comment:
In this case, x and y are not userdata, but are the defined by the callback API.